exercises/forth/forth_test.py

```diff
@@ -4,13 +4,30 @@
 
 
 # Tests adapted from `problem-specifications//canonical-data.json` @ v1.2.0
+# Tests for case-insensitivity are track-specific
```

Comment:
RE: [problem description](https://github.com/exercism/problem-specifications/blob/7735764b0882360563772aa66a2b1fd77c5bb568/exercises/forth/description.md):

> Words are case-insensitive.

Reply:
@cmccandless, the reason for the comment was that tests for case insensitivity were not in the canonical data, even if they were in the problem description. Perhaps I could have worded it better/differently, but I'm not sure that it is worth changing now?